test/tools/IsolTestOptions.cpp

```diff
@@ -56,14 +56,22 @@ std::string editorPath()
 
 }
 
-IsolTestOptions::IsolTestOptions(std::string* _editor):
+IsolTestOptions::IsolTestOptions():
 	CommonOptions(description)
 {
+	enforceViaYul = true;
+	enforceGasTest = (evmVersion() == langutil::EVMVersion{});
+	enforceGasTestMinValue = 100000;
```

Comment:
I just noticed that this value is the same as in `CommonOptions`. Could you remove it from here? The less places where defaults are hard-coded, the better.

Reply:
Done.